src/components/ServiceInstallation/Tabs/Services.vue

```diff
@@ -150,11 +160,21 @@
       },
       fetchServiceStatus() {
         this.getServices().then((response) => {
+          const servicesToBeRender = []
+
           this.lastFetched = new Date().toLocaleTimeString()
           const requiredServices = response.filter((service) =>
             this.servicesToCheck.includes(service.id)
           )
-          this.listOfServices = requiredServices
+          const requiredServicesTypeDb = response.filter((service) =>
+            this.servicesToCheckTypeDB.includes(service.id)
```

Comment:
you are using filter above right ? what is the purpose of a another filter

since you have servicesToCheck and filtering the, then again why servicesToCheckTypeDB is used to filter. 

Both have same contents the new one 2 new things right ?

Reply:
> you are using filter above right ? what is the purpose of a another filter
> 
> since you have servicesToCheck and filtering the, then again why servicesToCheckTypeDB is used to filter.
> 
> Both have same contents the new one 2 new things right ?


> you are using filter above right ? what is the purpose of a another filter
> 
> since you have servicesToCheck and filtering the, then again why servicesToCheckTypeDB is used to filter.
> 
> Both have same contents the new one 2 new things right ?

Using two arrays for different conditions

Reply:
Thats my question, you are using two array, the difference between the two arrays is just 2 data right ?

so you can directly have that 2 alone separately or use one filter loop for the one which have all the services, update the status based on the status of respective service

Reply:
Done